sdk/openai/ai-openai/package.json

```
@@ -0,0 +1,152 @@
+{
+  "name": "@azure/ai-openai",
+  "version": "1.0.0-beta.1",
+  "description": "An isomorphic client library for Azure OpenAI.",
+  "sdk-type": "client",
+  "main": "dist/index.js",
```

should this be `dist/index.cjs` since there is no `dist/index.js` generated? /cc @witemple-msft/@joheredi?
```suggestion
  "main": "dist/index.cjs",
```

---

IIRC if you are using `dev-tool run bundle` it will name the output bundle whatever is in this field, so `dist/index.js` or `dist/index.cjs` shouldn't matter. If the `type` of the package is `module` you may _need_ to set it to `dist/index.cjs` so that Node will load it as a CJS module. I'm still a little foggy on how all of this works until I sit down with a specific package and try it.